package.json

```diff
@@ -36,6 +36,7 @@
         "node-sass": "^4.5.0"
     },
     "scripts": {
+        "install": "bundle install",
```

Comment:
It means that when someone runs  `npm install`, it will also run `bundle install` to install the ruby dependencies

Reply:
not sure that's a good idea since people use `yarn` also :)

Reply:
it's the same. if you'll run `yarn install` it will run `bundle install` after that

I can remove this if you don't like the idea. I just thought it's nice to have all dependencies installed at the same time